bin/cmd/create-skin.js

```diff
@@ -17,8 +17,13 @@ module.exports = {
             return true;
         }
         var server = require('../../lib/server.js')(gulp, gutil);
+        var serverVersion = {
```

Comment:
I wonder if we should move this to the server-version.json file (using a key of some kind to match the script)?